samples/calendar/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
We generally don't include a separate `package.json` for individual samples like this, and rely on the higher level one.  I'm totally open to the idea, but for now I'd like to drop this for the sake of consistency.